### ISSUE TITLE
With version 1.0.1 the with and from sentences does not return throwable

### ISSUE THREAD

**Issue by watarhu** [bytemechanics/type-ex#4]
With version 1.0.1 the with and from sentences does not return java.lang.Throwable, therefore can not be thrown.
#### Example code:
```Java
throw CMBridgeExceptionType.ERROR_REPORTER_UNABLE_TO_COMPRESS_FILES.with(_files).from(e);
``` 
#### Causes:
> error: incompatible types: TypifiableException cannot be converted to Throwable

**Comment by watarhu** [comment]
The problem is that you can not compile a generic class that extends **java.lang.Throwable**, the reason explained at [Why doesn't Java allow generic subclasses of Throwable?](https://stackoverflow.com/questions/501277/why-doesnt-java-allow-generic-subclasses-of-throwable#501309), for this reason the only solution comes overriding the exception returning methods like:
```Java
	@Override
	public MockedCheckedException get() {
		return (MockedCheckedException)super.get();
	}
	@Override
	public MockedCheckedException from(final Throwable _cause) {
		return (MockedCheckedException)super.from(_cause); 
	}
	@Override
	public MockedCheckedException with(final Object... _args) {
		return (MockedCheckedException)super.with(_args); 
	}
```

**Comment by watarhu** [comment]
Finally the fixed source will come with release 1.0.2
